test/run/test_assemblers.py

```diff
@@ -220,11 +221,29 @@ def test_tar(self):
                 with self.run_assembler(osb,
                                         "org.osbuild.tar",
                                         options,
-                                        filename) as (_, image):
+                                        filename) as (tree, image):
                     output = subprocess.check_output(["file", "--mime-type", image], encoding="utf-8")
                     _, mimetype = output.strip().split(": ") # "filename: mimetype"
                     self.assertIn(mimetype, expected_mimetypes)
 
+                    if compression:
+                        continue
+
+                    # In the non-compression case, we verify the tree's content
+                    with tempfile.TemporaryDirectory(dir="/var") as tmp:
```

Comment:
```suggestion
                    with tempfile.TemporaryDirectory(dir="/var/tmp") as tmp:
```

Reply:
Well, `doh`, yes, obviously. That is what I have meant to type. Thanks!